stream.go

```diff
@@ -130,17 +130,17 @@ func (ds *DISTIL) StreamFromPath(path string) *Stream {
 }
 
 // Obtain multiple streams based on paths
-func (ds *DISTIL) StreamsFromPaths(paths []string) []*Stream {
+func (ds *DISTIL) StreamsFromPaths(paths []string) ([]*Stream, error) {
 	//loop over StreamFromPath
 	var streams = make([]*Stream, len(paths))
 	for i, path := range paths {
 		streams[i] = ds.StreamFromPath(path)
 		if streams[i] == nil {
-			fmt.Printf("could not locate stream %q\n", path)
-			os.Exit(1)
+			fmt.Printf("WARN: could not locate stream (%s)\n", path)
+			return nil, errors.New(fmt.Sprintf("Stream not found (%s)", path))
```

Comment:
Typically you use `fmt.Errorf` for this pattern
```suggestion
			return nil, fmt.Errorf("Stream not found (%s)", path)
```

Reply:
ahh interesting - I did not know about that!